tools/fixup-wheels.sh

```diff
@@ -117,9 +117,10 @@ fixup_wheel () {
         fixup_wheel_macosx
     fi
 
-    zip -r $fixed_whl .
-
     popd # tmpdir
+
+    python rebuild-wheel.py --staging-dir $tmpdir --wheel-file $fixed_whl
```

Comment:
I'm not sure what's the best way to define the path to the `rebuild-wheel.py` script. Should this be `tools/rebuild-wheel.py` or do you normally anchor the path based on the shell script location?

Reply:
I believe this script is run from the top-level project directory, so you can assume that being the case and reference rebuild-wheel.py through a relative path.